benchmarks/benchmarks.c

```diff
@@ -55,3 +54,188 @@ void benchmarkFillup ()
 		}
 	}
 }
+
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
+
+// needed for the opmphmRandom ()
+#include "../src/libs/elektra/opmphm.c"
```

Comment:
I am also not happy with this solution, I could rename `opmphmRandom ()` to `elektraRandom ()` then the function gets exported. But I think nobody except me needs a random function in the elektra core.